researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -430,24 +423,18 @@ public Completable forgotPassword(@NonNull String email) {
     @Override
     @Nullable
     public User getUser(@Nullable Context context) {
-        logger.debug("Called getUser");
-        if (userLocalStorage == null) {
-            return null;
-        }
-        return userLocalStorage.loadUser();
+        return researchStackDAO.getUser();
     }
 
     @Override
     @Nullable
     public void setUser(Context context, User user) {
-        logger.debug("Called getUser");
-        userLocalStorage.saveUser(user);
+        researchStackDAO.setUser(user);
     }
 
     @Override
     public String getUserEmail(Context context) {
-        User user = userLocalStorage.loadUser();
-        return user == null ? null : user.getEmail();
+        return authenticationManager.getEmail();
```

Comment:
I don't know.. should getUserEmail return the signed in user? or the user in the User object?

I don't understand the use case for storing User but added it to not regress what's currently in RS. AFAIK this didn't exist in MoleMapper or ResearchStack up until recently, User object was not used at all by RS before